src/tools/constants.rs

```diff
@@ -38,5 +38,11 @@ pub const ERR_SYNTH_UNICAST: &str = "unable to send a message";
 /// Error message when synthetic node creation fails.
 pub const ERR_SYNTH_BUILD: &str = "unable to build a synthetic node";
 
+/// Error message for a failed connection.
+pub const ERR_SYNTH_CONNECT: &str = "unable to connect to the node";
```

Comment:
I think we can delete this one:
`pub const ERR_NODE_CONNECT: &str = "unable to connect to the node";`

Reply:
Deleted for now.